docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -85,6 +85,10 @@ default::
 +
 [source,yaml]
 ----
+global:
+  mon_osd_full_ratio: 0.85
+  mon_osd_backfill_full_ratio: 0.8
+  mon_osd_nearfull_ratio: 0.75
```

Comment:
Since you've pointed out floating point rounding issues multiple times, shouldn't that be configured here as strings as well?
```suggestion
  mon_osd_full_ratio: '0.85'
  mon_osd_backfill_full_ratio: '0.8'
  mon_osd_nearfull_ratio: '0.75'
```

Reply:
Fixed in latest version